core/trino-main/src/test/java/io/trino/execution/TestSqlTaskManager.java

```diff
@@ -264,6 +265,10 @@ public void testFailStuckSplitTasks()
         // Here we explicitly enqueue an indefinite running split runner
         taskExecutor.enqueueSplits(taskHandle, false, ImmutableList.of(mockSplitRunner));
         taskExecutor.start();
+        // Wait for taskExecutor processed splits
```

Comment:
nit: `wait for the task executor to start processing the split`

Reply:
done